ajax/libs/hola_player/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "hola_player",
+  "description": "An HTML5 and Flash video player with a common API and skin for both.",
+  "version": "1.0.54",
+  "filename": "hola_player.js",
+  "license": "ISC",
+  "keywords": [
+    "flash",
+    "html5",
+    "player",
+    "video",
+    "videojs"
+  ],
+  "homepage": "http://holacdn.com/player",
```

Comment:
I'd like to.
But there is a chain issue on this website.
I don't know is it good to use `ssl`.
ref: https://www.ssllabs.com/ssltest/analyze.html?d=holacdn.com&s=184.73.238.209

@pvnr0082t could you suggest us?